test/list_resources/test_integration_list_role_members.py

```diff
@@ -0,0 +1,74 @@
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationTestList(IntegrationTestCaseBase):  # pragma: no cover
+    def __init__(self, test_name, client_params=None, environment_params=None):
+        super(CliIntegrationTestList, self).__init__(test_name, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setUp(self):
+        self.setup_cli_params({})
+        # Used to configure the CLI and login to run tests
+        utils.setup_cli(self)
+        self.invoke_cli(self.cli_auth_params,
+                        ['policy', 'replace', '-b', 'root', '-f',
+                         self.environment.path_provider.get_policy_path("list_role_members")])
+
+    # *************** TESTS ***************
+
+    @integration_test()
+    def test_list_members_of_single_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['list', '--members-of', 'group:devs'])
+        account = self.client_params.account
+        expected_result = "[\n" \
+                          f"  \"{account}:user:admin\",\n" \
+                          f"  \"{account}:user:alice\"," \
+                          f"  \"{account}:user:bob\"" \
+                          "]"
+        self.assertRegex(output, expected_result)
+
+    @integration_test()
+    def test_list_members_of_role_when_kind_is_not_specified_in_identifier(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['list', '-m', 'devs'])
+        account = self.client_params.account
+        expected_result = "[\n" \
+                          f"  \"{account}:user:admin\",\n" \
+                          f"  \"{account}:user:alice\"," \
+                          f"  \"{account}:user:bob\"" \
+                          "]"
+        self.assertRegex(output, expected_result)
+
+    @integration_test()
+    def test_list_members_of_role_missing_identifier_should_raise_error(self):
+        with self.assertRaises(AssertionError):
+            self.invoke_cli(self.cli_auth_params, ['list', '--members-of'])
+
+    @integration_test()
+    def test_list_members_of_role_missing_identifier_short_flag_should_raise_error(self):
+        with self.assertRaises(AssertionError):
+            self.invoke_cli(self.cli_auth_params, ['list', '-m'])
+
+    @integration_test()
+    def test_list_members_of_non_exist_role_should_raise_error(self):
+        with self.assertRaises(AssertionError):
+            self.invoke_cli(self.cli_auth_params,
+                            ['list', '--members-of', 'unknown_resource'])
+
+    @integration_test()
+    def test_list_member_of_role_when_role_kinds_have_same_id_should_raise_error(self):
+        with self.assertRaises(AssertionError):
+            self.invoke_cli(self.cli_auth_params, ['list', '--members-of', 'devs2'])
```

Comment:
You did not test the other flags you support, `--inspect`, `--limit` etc.
I think you should, since you pass them explicitly in your code.

Reply:
Added tests